### DISCUSSION TITLE
PWA no Google Doodle?

### DISCUSSION THREAD

**Discussion by mateuspntx** [frontendbr/forum#659] [Perguntas e Respostas]
Isto seria um Progressive Web App? Rodou como um aplicativo no Android. É uma maravilha.

![_20170811_132357](https://user-images.githubusercontent.com/8277449/29222384-a58cac94-7e98-11e7-8959-194f4c54987f.JPG)

**Comment by houstondapaz** [comment]
O meu não rodou, como vc fez?

**Comment by mateuspntx** [comment]
Apertei no play. Abriu uma janela em fullscreen com um "joguinho" dentro. Com apresentação e tutoriais.

**Comment by ngoctrai1997** [comment]
Progressive Web App vai melhorar muito e fazer muitas coisas serem melhores
Devemos aprender sobre isso
[https://tapita.io/](url)
